src/js/components/Diagram/stories/Progressing.js

```diff
@@ -28,57 +28,58 @@ const connection = (fromTarget, toTarget, { color, ...rest } = {}) => ({
 });
 
 const fullTopRow = [1, 2, 3];
-class SimpleDiagram extends React.Component {
-  state = { topRow: fullTopRow.slice(0, 1) };
 
-  componentDidMount() {
-    this.timer = setInterval(() => {
-      const { topRow } = this.state;
-      this.setState({
-        topRow: fullTopRow.slice(
-          0,
-          topRow.length < fullTopRow.length ? topRow.length + 1 : 1,
-        ),
-      });
+const SimpleDiagram = () => {
```

Comment:
```suggestion
const ProgressingDiagram = () => {
```
Renaming the component to use a proper name that is aligned with the story.

Reply:
Oh sure, thanks for noticing :+1: